service/qos/service_level_controller.cc

```diff
@@ -548,6 +549,25 @@ future<> service_level_controller::do_add_service_level(sstring name, service_le
     return make_ready_future();
 }
 
+service_level_map service_level_controller::get_service_levels(const bool with_static, const bool with_marked_for_deletion) const {
```

Comment:
This function always seems to be called with `with_static == false`. Do we need this parameter at all?

Reply:
I introduced it to make the function reusable in the future